### PR TITLE
SourceLink added to all projects

### PR DESCRIPTION
**Pull request by BruceHaley**
This sets symbol references to the GitHub source location. In a debug session, source is automatically downloaded when needed from GitHub.

### CODE REVIEW

**Comment by cleemullins** [comment]
@tomlm - Can you take a look and approve before we merge it? I think this is good, but I may be missing something as the v3 SDK has nuances of which I'm unaware...

File: CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config [hunk 1]
```diff
@@ -87,6 +87,10 @@
         <assemblyIdentity name="Microsoft.Azure.Documents.Client" publicKeyToken="31bf3856ad364e35" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-1.11.0.0" newVersion="1.11.0.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Net.Http" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.0.0" newVersion="4.0.0.0" />
+      </dependentAssembly>
```
Comment (willportnoy) [inline CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config]: risky?

**Comment by willportnoy** [comment]
Does this add runtime dependencies to the assemblies or package dependencies to NuGet?

---
In reply to: [401152931](https://github.com/Microsoft/BotBuilder/pull/4808#issuecomment-401152931) [](ancestors = 401152931)

File: CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config (hunk 1 above)
Comment (BruceHaley) [reply CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config]: Package Manager did this. Not sure of the ramifications.

Comment (BruceHaley) [reply CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config]: SourceLink  creates no runtime dependencies or package dependencies that I know of. It's just that symbols have links to source in GitHub. Tracing during debug then automatically downloads the source as needed.

Comment (BruceHaley) [reply CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config]: Bot Builder V4 already has this implemented.

Comment (BruceHaley) [reply CSharp/Samples/Microsoft.Bot.Sample.FormFlowAttachmentsBot/Web.config]: The SourceLink.Create.CommandLine NuGet package gets imported in the project builds.

**Pull request merged**